docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+## Overview
+This file contains general contributing information, but individual parts also may have more specific information to them, linked here:
+    - ``[FFBEffects-T300RS](./FFBEffects-T300RS.md)``: Force Feedback Effects example for T300RS
+    - ``[FFBEffects-T248](./FFBEffects-T248.md)``: Force Feedback Effects example for T300RS
+    - ``[To-Do](./TODO.md)``: TO-DO list for maintainers
+    - ``[Structure](https://github.com/Kimplul/hid-tmff2/wiki#structure-of-the-thrustmaster-device-stack)``: Structure of Thrustmaster device stack
+
+## How to add in support for a new T-series wheel?
+Should probably not be too often that you need this info, but essentially use wireshark like in the previous example, but spin up a Windows virtual machine and install the Thrustmaster drivers on to it, and pass the device to the virtual machine. I prefer to use ``qemu`` with ``virt-manager`` as a frontend. With the wheel working under Windows, install [fedit.exe](https://gimx.fr/download/b882e209a0ac023d03abbf560dfc3f25fe6367ca/fedit.zip) and methodically go through all effects the device supports and compare the USB packets the driver sends out. You should be able to build up a table of what each value in the USB packet means, see [FFBEffects-T300RS.md](./FFBEffects-T300RS.md) for an example of what I found out about the T300.
+
+## How to capture what effects a game sends to the driver?
+Use ffbwrap from ffbtools: [github:berarma/ffbtools](https://github.com/berarma/ffbtools)
+The documentation gives good examples, but tl;dr; For steam, insert the following into a game's launch options:
+```shell
+$ ffbwrap --logger=/home/$USER/game.log /dev/input/by-id/usb-Thrustmaster_Thrustmaster_T300RS_Racing_wheel-event-joystick -- %command%
+```
+This will create a file called ``game.log`` (with additional timestamp) in your home directory. Preferably change the name to suit the game, but you do you.
```

Comment:
Any particular reason to prefer double backticks over single backticks? I understand it escapes single backticks, but that seems overkill in this situation.

Reply:
There is no special reason. I see that the `README.md` uses single backticks so, I have switched the double backticks to single ones in https://github.com/Kimplul/hid-tmff2/pull/73/commits/c5e36a43b975177f37d9c3bf42ed657c44cbd5d5